ledger-core/v2/conveyor/smachine/api_shared_link.go

```diff
@@ -8,10 +8,23 @@ package smachine
 import (
 	"fmt"
 	"reflect"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/throw"
 )
 
 type SharedDataFunc func(interface{}) (wakeup bool)
 
+func NewUnboundSharedData(data interface{}) SharedDataLink {
```

Comment:
What for?

Reply:
I mean this function is not used anywhere

Reply:
This function create SharedDataLink which sometimes need for tests. But unit test was deleted (it does not work). Do i must remove this function too?

Reply:
Yes, it does not used right now. I can remove it. But maybe it useful for future tests?

Reply:
remove this code